samples/net/cloud/aws_iot_mqtt/src/tls_config/user-tls.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Lucas Dietrich <ld.adecy@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define MBEDTLS_AES_ROM_TABLES
+
+#define MBEDTLS_HAVE_TIME
+#define MBEDTLS_HAVE_TIME_DATE
+#define MBEDTLS_PLATFORM_TIME_ALT
+
+#undef MBEDTLS_SSL_KEEP_PEER_CERTIFICATE
```

Comment:
Why is this needed? The generic mbedTLS config file from Zephyr does not set this config.